api/resources_portal/config/common.py

```diff
@@ -195,5 +194,9 @@ class Common(Configuration):
         "guardian.backends.ObjectPermissionBackend",
     )
 
+    # Time for expiration of API tokens
+    # About 6 months
+    EXPIRING_TOKEN_DURATION = timedelta(weeks=26)
```

Comment:
Any reason not to just use `timedelta(months=6)`?

Reply:
timedelta's largest available time interval is weeks.